scss/foundation_dependencies.scss

```diff
@@ -25,7 +25,6 @@
 
 
 // Imports
-@include foundation-typography;
 @include foundation-visibility-classes;
 
 {{#components}}
```

Comment:
Had the thought looking at this... why are we including so many components? Almost certainly we only need a few...